### DISCUSSION TITLE
I want to add a sprite trail for speed shoes

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#66] [Q&A]
I am trying to add a sprite trail for speed shoes how would I do it

**Comment by DimensionWarped** [comment]
You've got a lot of options... and if I were doing it personally, I wouldn't even use the speed shoes but instead just while being at high speed values.

A - use a shader -- You can probably find a motion blur shader around the Internet and you could either apply it directly to the player sprite or better yet to a copy of the player sprite lower in the layer order to get an effect like what you want while at high speed.
B - Draw copies of the sprite with transparency and optional color effects (pretty easy, just look into how to use the Draw function. You can probably find similar examples of what you are looking for with the way invincibility stars are already being drawn.

If what you want is more like speed lines behind the player you can look into particle systems too.

Sorry to say I can't give you more detailed implementation advice than that right now.
